.github/workflows/build.yml

```diff
@@ -60,14 +60,15 @@ jobs:
       fail-fast: true
       matrix:
         os: ['ubuntu-16.04', 'ubuntu-18.04']
+        python-version: ['3.6','3.7','3.8']
```

Comment:
Update the builder name with the Python version too.

Reply:
> Update the builder name with the Python version too.

I did...then I still had lines that were too long :D